tests/einstein/operations/unit/test_operations.py

```diff
@@ -1,40 +1,48 @@
+# Tests the calls in einstein/operations.py
+
 import time
 import unittest
 from unittest import TestCase
 
+import ims.common.config as config
 import ims.common.constants as constants
 import ims.einstein.ceph as ceph
+from ims.common.log import trace
+from ims.database.database import Database
 from ims.einstein.operations import BMI
 
 _cfg = config.get()
 
-CORRECT_HAAS_USERNAME = "haasadmin"
-CORRECT_HAAS_PASSWORD = "admin1234"
-INCORRECT_HAAS_PASSWORD = "admin123##"
+CORRECT_HIL_USERNAME = _cfg.tests.correct_hil_username
+CORRECT_HIL_PASSWORD = _cfg.tests.correct_hil_password
+INCORRECT_HIL_PASSWORD = _cfg.tests.incorrect_hil_password
 
-NODE_NAME = "cisco-24"
-NIC = "enp130s0f0"
+NODE_NAME = _cfg.tests.node_name
+NIC = _cfg.tests.nic
 
-PROJECT = "bmi_infra"
-NETWORK = "bmi-provision"
+PROJECT = _cfg.tests.project
+NETWORK = _cfg.tests.network
 
-EXIST_IMG_NAME = "bmi_ci.img"
-NEW_SNAP_NAME = 'test_snap'
-NOT_EXIST_IMG_NAME = "i12"
-NOT_EXIST_SNAP_NAME = "hello"
+EXIST_IMG_NAME = _cfg.tests.exist_img_name
+NEW_SNAP_NAME = _cfg.tests.new_snap_name
+NOT_EXIST_IMG_NAME = _cfg.tests.not_exist_img_name
+NOT_EXIST_SNAP_NAME = _cfg.tests.not_exist_snap_name
 
```

Comment:
So this test would work if we have a real setup or if we have mock drivers. So maybe we should put a comment saying that it is not a unit test and will not work. 
And if we want to keep this the way it is right now, we could put it under `/tests/integration`, you get what I am saying?